plugins/@grouparoo/pipedrive/public/templates/destination/{{{id}}}.js.template

```diff
@@ -4,7 +4,7 @@ exports.default = async function buildConfig() {
       id: {{{id}}},
       name: {{{id}}},
       class: "destination",
-      type: "{{{__pluginName}}}-export",
+      type: "pipedrive-export-persons",
```

Comment:
While an interesting grammatical choice, that is what pipedrive calls the people